contracts/veBalFeeInjector.sol

```diff
@@ -75,16 +81,16 @@ contract veBalFeeInjector is ConfirmedOwner, Pausable {
    *
    */
   function performUpkeep(bytes calldata performData) external  onlyKeeperRegistry whenNotPaused {
-    bool upkeepNeeded;
+    uint256 timeCursor = feeDistributor.getTimeCursor();
+    require(LastRunTimeCurser < timeCursor, "Not ready");
```

Comment:
Moved the time curser check from _pay_fees to performUpkeep() so admin can pay fees a second time in an epoch without having to sweep the contract and do so manually.